src/structure.ts

```diff
@@ -53,26 +53,26 @@ export interface Components<Ctx extends Context = Context, Req extends Request =
 /**
  * Validate that a given value is a {@link Command} object
  */
-const isCommand = (value: any, warn = false): value is Command => {
+const isCommand = (value: unknown, warn = false): value is Command => {
     if (typeof value !== 'object' || value === null) {
         if (warn)
             console.warn('Expected command to be an object');
         return false;
     }
 
-    if (typeof value.name !== 'string' || !value.name.length) {
+    if (!('name' in value) || typeof value.name !== 'string' || !(value.name.length)) {
```

Comment:
```suggestion
    if (!('name' in value) || typeof value.name !== 'string' || !value.name.length) {
```
Parens here are redundant?